src/fullnode.rs

```diff
@@ -24,7 +24,7 @@ use std::net::UdpSocket;
 use std::net::{IpAddr, Ipv4Addr, SocketAddr};
 use std::sync::atomic::{AtomicBool, AtomicUsize, Ordering};
 use std::sync::mpsc::channel;
-use std::sync::mpsc::{Receiver, Sender};
+use std::sync::mpsc::{Receiver, Sender, SyncSender};
```

Comment:
The SyncSender here allows us to implement a channel of capacity one, so that when the db_ledger is sending a signal for an update: https://github.com/solana-labs/solana/pull/2597/files#diff-778eca2f590e4997d830337a4076d5ebR578, it can just try_send and move on. On the flip side, when the replay stage is woken up by this channel, it doesn't need to clear/dequeue everything from the channel by calling recv() multiple times, instead it only needs to pull the one update.